sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosClientCache.scala

```diff
@@ -260,6 +260,10 @@ private[spark] object CosmosClientCache extends BasicLoggingTrait {
               SparkBridgeImplementationInternal
                   .setIoThreadPriority(directConfig, Thread.MAX_PRIORITY)
 
+          directConfig =
```

Comment:
Let's discuss tomorrow - I think I understand why you want to be careful here - but want to double-check...

Reply:
Discussed - all good.